File: backend/rs-client/php/src/service/ThirdPartyCreativeService.php
<?php

/**
 
 */
class ThirdPartyCreativeService {

    private $configuration;

    public function __construct(RsConfiguration $configuration) {
        $this->configuration = $configuration;
    }

    
    public function get (array $thirdPartyCreativeSelector) {
        $client = $this->configuration->createClient();
        $parameters = null;
        $data = null;
    
        $parameters = $thirdPartyCreativeSelector;
        
    
        return $client
                ->request("GET" ,"/thirdPartyCreatives/", $parameters, $data);
    }
    
    public function perform ($operations) {
        $client = $this->configuration->createClient();
        $parameters = null;
        $data = null;
    
        
        $data = $operations;
    
        return $client
                ->request("POST" ,"/thirdPartyCreatives/", $parameters, $data);
    }
}

File: backend/rs-client/java/src/main/java/com/foros/rs/client/service/ThirdPartyCreativeService.java
package com.foros.rs.client.service;

import com.foros.rs.client.RsClient;
import com.foros.rs.client.data.JAXBEntity;
import com.foros.rs.client.model.siteCreative.ThirdPartyCreative;
import com.foros.rs.client.model.siteCreative.ThirdPartyCreativeOperations;
import com.foros.rs.client.model.siteCreative.ThirdPartyCreativeSelector;

public class ThirdPartyCreativeService extends ReadonlyServiceSupport<ThirdPartyCreativeSelector, ThirdPartyCreative> {

    public ThirdPartyCreativeService(RsClient rsClient) {
        super(rsClient, "/thirdPartyCreatives");
    }

    public void perform(ThirdPartyCreativeOperations operations) {
        rsClient.post(path, new JAXBEntity(operations));
    }
}